### PR TITLE
Migrate most compat_keys from web-features

### PR DESCRIPTION
**Pull request by foolip**
This was done using the script in https://github.com/web-platform-dx/web-features/pull/473
with a few local changes to exclude features that are already tagged in
BCD and will need a closer look to ensure everything is in sync:

- array-at
- class-syntax
- container-queries
- font-variant-alternates
- scroll-driven-animations
- svg2-script-html-equivalence

The slot feature is also excluded to avoid conflicts:
https://github.com/mdn/browser-compat-data/pull/22781

In other words, this change is the simple cases, only new tags that are not
anywhere in BCD yet.

### CODE REVIEW

File: svg/elements/script.json [hunk 1]
```diff
@@ -147,7 +147,8 @@
             "__compat": {
               "description": "<code>type='module'</code>",
               "tags": [
-                "web-features:svg2-script-html-equivalence"
```
Comment (foolip) [inline svg/elements/script.json:150]: This is the only removal in this diff, but it's because of the comma. This might also be the only entry tagged with two features.

**Review by Elchi3 — APPROVED** [review]
LGTM, especially api/Document.json was a joy to look at. Great seeing various features tagged. I'm happy to merge this unless you need to coordinate this PR with something over in web-features?

**Comment by foolip** [comment]
I'll merge this now to avoid merge conflicts accumulating.
